unittests/DefToYAMLConverter/DefToYAMLConverterTest.cpp

```diff
@@ -60,22 +76,37 @@ TEST(DefToYAMLConverterTest, missingLocalizationFiles) {
 }
 
 TEST(DefToYAMLConverterTest, matchDiagnosticMessagesSequentially) {
-  llvm::SmallString<128> EnglishLocalization(getDefaultLocalizationPath());
-  llvm::sys::path::append(EnglishLocalization, "en");
-  llvm::sys::path::replace_extension(EnglishLocalization, ".yaml");
-  YAMLLocalizationProducer yaml(EnglishLocalization.str());
+  llvm::SmallString<128> defYAMLConverter(getDefToYAMLConverterPath());
+  defYAMLConverter.append(" --output-filename=");
+
+  llvm::SmallString<128> tempFilename;
+  std::error_code EC =
+      llvm::sys::fs::createTemporaryFile("en", "yaml", tempFilename);
+  ASSERT_FALSE(EC);
+  llvm::sys::RemoveFileOnSignal(tempFilename);
+  defYAMLConverter.append(tempFilename);
+  std::system(defYAMLConverter.c_str());
```

Comment:
@xedin  Is `std::system` is the right method for this? I tried looking for something similar in `llvm` but wasn't able to find anything like it.

Reply:
This is not exactly what I meant - converter should be property abstracted which would allow it to be called directly just like `YAMLLocalizationProducer`

Reply:
Fixed!